src/Sign.Cli/AzureAuthOptions.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE.txt file in the project root for more information.
+
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using Azure.Core;
+using Azure.Identity;
+using Sign.Core;
+
+namespace Sign.Cli
+{
+    internal sealed class AzureAuthOptions
+    {
+        internal Option<bool> ManagedIdentityOption { get; } = new(["-azm", "--azure-managed-identity"], getDefaultValue: () => false, Resources.ManagedIdentityOptionDescription);
```

Comment:
I think some of these descriptions mention Key Vault but these don't make sense when the options are used in the `TrustedSigningCommand`

Reply:
Thanks for finding that. I will update the documentation once it is clear that this new `AzureAuthOptions` class is a move in the right direction.

Reply:
I have updated the documentation.